deploy.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+# Deployment script for kube - generates configmaps from SQL init files and provisions everything
+
+GREEN="\033[1;32m"
+BLUE="\033[1;34m"
+NOCOLOR="\033[0m"
+
+minikube start --vm-driver=virtualbox
+
+echo ${GREEN}
+
+kubectl create secret docker-registry regcred --docker-server=$REG_URL --docker-username=$REG_USERNAME --docker-password=$REG_PASSWORD --docker-email=$REG_EMAIL
+kubectl create configmap match-db-config --from-file match-db/init.sql -o=yaml
+kubectl create configmap user-db-config --from-file user-db/init.sql -o=yaml
+kubectl create configmap auth-db-config --from-file auth-db/init.sql -o=yaml
+kubectl create -f kube/kong
+kubectl create -f kube/user
+kubectl create -f kube/match
+kubectl create -f kube/auth
```

Comment:
`ls -d kube/* | xargs -I"@" kubectl create -f @` if you prefer 😉

Reply:
Jk this probably isn't as readable

Reply:
I'll do this as a for loop instead